### PR TITLE
Added notification message object to store notification in one more p…

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
…lace

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nthomas-redhat please review

**Comment by GowthamShanmugam** [comment]
verified

File: tendrl/node_agent/objects/notification_message/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class NotificationMessage(objects.BaseObject, CommonMessage):
+    internal = True
+
+    def __init__(self, **notification_message):
+        self._defs = {}
+        CommonMessage.__init__(self, **notification_message)
+        objects.BaseObject.__init__(self)
+
+        self.value = '_tendrl/notifications/{0}'
```
Comment (r0h4n) [inline tendrl/node_agent/objects/notification_message/__init__.py]: Put this in /notifications/ and not _tendrl/notifications

Comment (GowthamShanmugam) [reply tendrl/node_agent/objects/notification_message/__init__.py]: ok i will change this

**Pull request merged**